websites/V/Videoland/presence.ts

```diff
@@ -0,0 +1,70 @@
+const presence = new Presence({
+		clientId: "941627291304329226"
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+let search: HTMLInputElement;
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+			largeImageKey: "logo",
+			startTimestamp: browsingTimestamp
+		},
+		page = window.location.pathname;
+
+	if (page === "/") presenceData.details = "Bekijkt de homepagina";
+	else if (page.includes("zoeken")) {
+		search = document.querySelector("#search");
+		presenceData.smallImageKey = "search";
+		if (search.value) {
+			presenceData.details = "Zoekt voor:";
+			presenceData.state = search.textContent;
+		} else presenceData.details = "Aan het zoeken...";
+	} else if (page.includes("/films/")) {
+		presenceData.details = document.querySelector<HTMLMetaElement>(
+			"meta[property='og:title']"
+		).content;
+	} else if (page.includes("/series/")) {
+		presenceData.details = document.querySelector<HTMLMetaElement>(
+			"meta[property='og:title']"
+		).content;
+	} else if (page.includes("/player/")) {
+		delete presenceData.startTimestamp;
+		const f = await fetch(
+			`https://www.videoland.com/api/v3/titles/${
+				window.location.href.split("/")[4]
+			}/details`
+		).then(x => x.json());
+		presenceData.details = await f.showTitle;
```

Comment:
yea maybe dont do this in updatedata... 
and cache/save previously fetched ones

Reply:
also more appropriate variable name